Sources/SpelledPitch/PitchSpeller/Wetherfield/UndirectedGraphSchemeProtocol.swift

```diff
@@ -8,3 +8,24 @@
 import DataStructures
 
 public protocol UndirectedGraphSchemeProtocol: GraphSchemeProtocol where Edge == UnorderedPair<Node> { }
+
+extension UndirectedGraphSchemeProtocol where Self: WeightedGraphSchemeProtocol {
+    
+    func directed <D> () -> D where
+        D: DirectedGraphSchemeProtocol & WeightedGraphSchemeProtocol,
```

Comment:
Are there any other types other than `WeightedDirectedScheme` which conform to both `DirectedGraphSchemeProtocol & WeightedGraphSchemeProtocol`

Reply:
Not at present, no! Would you rather these lived in structs as computed properties?

Reply:
By which I mean, this would live in `WeghtedGraphScheme` and map to `WeightedDirectedGraphScheme`...

Reply:
I'm split. I think symmetry with the `UnweightedGraphProtocol.unweighted()` is nice (being a function and all).

However, it might make it easier to reason about if we stick to concrete types.

Reply:
Oh, I think that would be optimal!

Reply:
Totally!

Reply:
(As source and destination are both effectively concrete types).

Reply:
👍